src/driver/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["The Rust Project Developers"]
+name = "driver"
+version = "0.0.0"
+
+[[bin]]
+name = "rustc"
+path = "rustc.rs"
+
+[[bin]]
+name = "rustdoc"
+path = "rustdoc.rs"
+
+[dependencies]
+rustc_driver = { path = "../librustc_driver" }
+std = { path = "../libstd" }
+rustdoc = { path = "../librustdoc" }
+
+[profile.release]
+opt-level = 3
+lto = true
+
+[profile.dev]
+opt-level = 1
```

Comment:
I don't think we'll want this to stay at O1 for dev and we probably also don't want to turn on LTO just yet. I also believe that opt-level 3 is the default for release so these two sections can be omitted.